quickstart/dotnet-core/README.md

```diff
@@ -34,7 +34,7 @@ dotnet restore
 Execute the following command to store a new secret named *CacheConnection*, after replacing the placeholders (including angle brackets) for your cache name and primary access key:
 
 ```
-dotnet user-secrets set CacheConnection "<cache name>.redis.cache.windows.net,abortConnect=false,ssl=true,password=<primary-access-key>"
+dotnet user-secrets set CacheConnection "<cache name>.redis.cache.windows.net,abortConnect=false,ssl=true,allowAdmin=true,password=<primary-access-key>"
```

Comment:
One other small question that can probably be left for a follow-up update is whether it would be useful to move the example code for calling `ClientList()` into a non-quickstart sample so `allowAdmin=true` could be removed from the quickstarts to keep them a little simpler?

According to the [StackExchange.Redis docs](https://stackexchange.github.io/StackExchange.Redis/Configuration.html), `allowAdmin=true` "Enables a range of commands that are considered risky," so maybe omitting it from the quickstarts would also help new developers discover which commands are "risky?"

Reply:
/cc @mapalan, in case you have any thoughts on this `allowAdmin=true` question. Thanks!

Reply:
I chatted with mapalan about this, and we agreed it would probably make sense to move `allowAdmin=true` out of the quickstarts in the future since the commands that require `allowAdmin=true` are not used too often. I'll add a work item to the backlog for that.